packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/InputStatus/InputStatus.js

```diff
@@ -1,29 +1,37 @@
-import React from 'react'
 import PropTypes from 'prop-types'
+import React from 'react'
+import { Text } from 'grommet'
 import styled from 'styled-components'
+import counterpart from 'counterpart'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
 
 export const StyledInputStatus = styled.span`
   font-size: 0.8em;
   opacity: 0.7;
   text-align: right;
 `
 
-// TODO Add Translations
 export default function InputStatus ({ count, tool }) {
   const minStyleColor = (count < tool.min) ? 'red' : ''
   const maxStyleColor = (count === tool.max) ? 'orange' : ''
+  const drawn = counterpart('InputStatus.drawn')
+  const maximum = counterpart('InputStatus.maximum')
+  const ofTranslation = counterpart('InputStatus.of')
+  const required = counterpart('InputStatus.required')
```

Comment:
This assumes the sentence structure will be `x of y required` in another language. I'm not sure if that's true.

Reply:
Perhaps the whole sentence should be processed by counterpart with variable placeholders for the counts?

Reply:
That would be my preferred option. I'm trying to remember why we didn't do that in PFE. I'm assuming because of the styling of the coloured text, and the conditional logic that constructs the sentence in English.

Reply:
Oh, and the logic here is complicated enough to consider moving this message into its own, testable, functional component.

Reply:
The color styling looks bad with the active teal color state as is, so let's maybe drop the colors and better support translations instead.

Reply:
Sounds good to me. Markdown strings might work better here anyway.

Reply:
Not sure what I'm doing wrong with the following `en` json translation object:
```
{
  "InputStatus": {
    "drawn": "%(count) drawn"
  }
}
```
Then attempting to set the status as follows (though won't be hardcoded to 10, simplifying troubleshooting):
```
counterpart.registerTranslations('en', en)
const status = counterpart('InputStatus.drawn', { count: '10' })
```
I get console error of:
```
Uncaught SyntaxError: [sprintf] unexpected placeholder
```

Reply:
This trips me up every time. Counterpart is expecting a syntax of `%(variable)s` note the `s` at the end.

Reply:
sprintf requires an `s` to specify a string, I think: `%(string)s`.

Reply:
ah ha thank you both!